models/articles.models.js

```diff
@@ -27,4 +27,31 @@ function fetchArticles() {
     })
 }
 
-module.exports = {fetchArticleById, fetchArticles}
\ No newline at end of file
+function updateArticleById(id, votes) {
+    if (typeof votes !== 'number') {
+        return Promise.reject({status:400, message: "Bad request"})
+    }
+    return db.query(
+        `SELECT votes
+        FROM articles
+        WHERE article_id = $1;`, [id])
+    .then(({rows}) => {
+        let currentVotes = rows[0].votes
+        let totalVotes = currentVotes += votes
```

Comment:
You could do this in the SQL with `SET votes = votes + $1` this would save you a database request